CondFormats/EcalObjects/interface/alpaka/EcalElectronicsMappingDevice.h

```diff
@@ -0,0 +1,17 @@
+#ifndef CondFormats_EcalObjects_interface_alpaka_EcalElectronicsMappingDevice_h
+#define CondFormats_EcalObjects_interface_alpaka_EcalElectronicsMappingDevice_h
+
+#include "CondFormats/EcalObjects/interface/EcalElectronicsMappingHost.h"
+#include "CondFormats/EcalObjects/interface/EcalElectronicsMappingSoA.h"
+#include "DataFormats/Portable/interface/alpaka/PortableCollection.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  using EcalElectronicsMappingHost = EcalElectronicsMappingHost;
```

Comment:
I think this should be
```suggestion
  using EcalElectronicsMappingHost = ::EcalElectronicsMappingHost;
```